### ISSUE TITLE
Specifying host "code.local" causes crashing

### ISSUE THREAD

**Issue by jcalfee** [solidjs/solid-start#1054]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

```bash
$ HOST=code.local npm run dev

 version  0.3.5
 adapter  node

  VITE v4.4.9  ready in 499 ms

  ➜  Network: http://code.local:3037/
  ➜  Inspect: http://localhost:3037/__inspect/
  ➜  press h to show help
file:///home/james/dev/ipall/ssweb/node_modules/.pnpm/solid-start@0.3.5_@solidjs+meta@0.28.6_@solidjs+router@0.8.3_solid-js@1.7.11_solid-start-node@0.3.5_vite@4.4.9/node_modules/solid-start/vite/plugin.js:243
              printUrls(config.solidOptions.router, url.substring(0, url.length - 1));
                                                        ^

TypeError: Cannot read properties of undefined (reading 'substring')
    at Timeout._onTimeout (file:///home/james/dev/ipall/ssweb/node_modules/.pnpm/solid-start@0.3.5_@solidjs+meta@0.28.6_@solidjs+router@0.8.3_solid-js@1.7.11_solid-start-node@0.3.5_vite@4.4.9/node_modules/solid-start/vite/plugin.js:243:57)
    at listOnTimeout (node:internal/timers:564:17)
    at process.processTimers (node:internal/timers:507:7)

Node.js v18.12.1
```

```js
import solid from "solid-start/vite"
import { defineConfig } from "vite"

export default defineConfig({
  plugins: [solid()],
  server: {
    host: process.env.HOST ?? 'localhost',
    port: process.env.PORT ?? 3037
  }
})
```

### Expected behavior 🤔

No crashing and working

### Steps to reproduce 🕹

Steps:

1. see above

**Comment by emdede** [comment]
Should be fixed after #1057. Can you check it with latest version (0.3.6)?

**Comment by jcalfee** [comment]
Confirmed, it works. Thank you!
